src/functions/pollUsers/framework/repo/dynamodb/cached-examiner-repository.ts

```diff
@@ -34,21 +35,21 @@ export const getCachedExaminers = async (): Promise<string[]> => {
   return scanResult.Items.map(item => item.staffNumber);
 };
 
-export const cacheStaffNumbers = async (staffNumbers: string[]): Promise<void> => {
-  console.log(`Caching ${staffNumbers.length} staff numbers...`);
+export const cacheStaffDetails = async (staffDetail: StaffDetail[]): Promise<void> => {
+  console.log(`Caching ${staffDetail.length} staff numbers...`);
   const ddb = getDynamoClient();
   const tableName = config().usersDynamodbTableName;
 
   const maxBatchWriteRequests = 25;
-  const staffNumberWriteBatches = chunk(staffNumbers, maxBatchWriteRequests);
+  const staffDetailWriteBatches: StaffDetail[][] = chunk(staffDetail, maxBatchWriteRequests);
 
-  const writePromises = staffNumberWriteBatches.map((batch) => {
+  const writePromises = staffDetailWriteBatches.map((batch) => {
     const params = {
       RequestItems: {
-        [tableName]: batch.map(staffNumber => ({
+        [tableName]: batch.map(staffDetail => ({
           PutRequest: {
             Item: {
-              staffNumber,
+              staffDetail,
```

Comment:
This is not the datastructure you want to persist. This would put the following in DDB:
```
{
  staffDetail: {
    staffNumber: '123',
    isLDTM: false
  }
}
```

You should do:
```
PutRequest: {
  Item: staffDetail,
},
```